_layouts/base.html

```diff
@@ -3,6 +3,7 @@
   <head>
     <meta charset="UTF-8">
     <title>{{ page.title }}</title>
+    {% if page.description %}<meta name="description" content="{{ page.description }}"/>{% endif %}
```

Comment:
This HTML5, so no empty element syntax.

```suggestion
    {% if page.description %}<meta name="description" content="{{ page.description }}">{% endif %}
```